client/src/main/java/org/eclipse/hono/client/impl/AbstractRequestResponseClient.java

```diff
@@ -371,10 +372,6 @@ private final void sendRequest(final Message request, final Handler<AsyncResult<
                 final Object correlationId = Optional.ofNullable(request.getCorrelationId()).orElse(request.getMessageId());
                 replyMap.put(correlationId, resultHandler);
                 sender.send(request, deliveryUpdated -> {
-
-                    // settle locally
-                    deliveryUpdated.settle();
```

Comment:
That should be removed right here.

Reply:
Oh, my bad, the eyes are getting worse and worse with the years ;-)